### PR TITLE
Describe "Unit Config" and "Customizing Size"

### PR DESCRIPTION
**Pull request by kanitw**
Now that I just finished writing docs — I’m having a second thought on `config.unit.*`(edited)
Maybe the styling of unit could be `config.cell.*` while `unitWidth` and `unitHeight` can be a top-level property because they are so important.

This way, we don't need a separate page for "Customizing Size".   
It could be a section called "Size Configuration" inside config.md.   

Another side question is I 'm thinking that we should split config.mark.md to its own file.

### CODE REVIEW

File: site/docs/config.md [hunk 1]
```diff
@@ -50,24 +50,23 @@ A Vega-Lite `config` object can have the following top-level properties:
...
-
-### Unit Size Configuration
+`width` and `height` property of the cell configuration determine the width of a visualization with a continuous x-scale and the height of a visualization with a continuous y-scale respectively.  
 
 | Property      | Type          | Description    |
 | :------------ |:-------------:| :------------- |
-| width         | Integer       | The width of the unit visualization (200 pixels by default).  This property is used only when `x` uses non-ordinal scale.  When `x` has an ordinal scale, the width is determined by x-scale's [`bandWidth`](scale.html#ordinal).  |
-| height        | Integer       | The height of the visualization for a single cell (200 pixels by default).  This property is used only when `y` has a non-ordinal scale.  When `y` has an ordinal scale, the height is determined by y-scale's `bandWidth`. |
+| width         | Integer       | The width of the unit visualization when the visualization has continuous x-scale.  (If the visualization has ordinal x-scale, the width is determined by the x-scale's [`bandWidth`](scale.html#ordinal) and the cardinality of the x-scale.) <span class="note-line">__Default value:__ `200`</span> |
```
Comment (domoritz) [inline site/docs/config.md]: still says unit here

**Comment by domoritz** [comment]
It still says unit in a lot of places. Please refer to cell instead.

**Comment by kanitw** [comment]
Done

File: site/docs/scale.md [hunk 2]
```diff
@@ -117,7 +117,7 @@ TODO: Custom Domain for quantitative
 The range of the scale represents the set of output visual values.  Vega-Lite automatically determines appropriate range based on the scale's channel and type, but `range` property can be provided to customize range values.  
 
 `x` and `y` Scales
-: For continuous `x` and `y` scales (quantitative and time), the range are always `[0, unitWidth]` and  `[0, unitHeight]` (See [config.unit](config.html#unit) for customizing unit width and height).  For ordinal `x` and `y` scales, the maximum range is a product of the field's cardinality and [`bandWidth`](#ordinal).  
+: For continuous `x` and `y` scales (quantitative and time), the range are always `[0, cellWidth]` and  `[0, cellHeight]` (See [config.cell](config.html#cell-config) for customizing unit width and height).  For ordinal `x` and `y` scales, the maximum range is a product of the field's cardinality and [`bandWidth`](#ordinal).
```
Comment (domoritz) [inline site/docs/scale.md]: says unit

File: src/compile/facet.ts [hunk 3]
```diff
@@ -100,8 +100,8 @@ function getFacetGroupProperties(model: Model) {
   };
 
   // apply both config from unit and facet.unit (with higher precedence for facet.unit)
```
Comment (domoritz) [inline src/compile/facet.ts]: unit

File: src/compile/scale.ts [hunk 4]
```diff
@@ -333,11 +333,11 @@ export function rangeMixins(scale: Scale, model: Model, channel: Channel, scaleT
 
       return {
         rangeMin: 0,
-        rangeMax: model.config().unit.width // Fixed unit width for non-ordinal
+        rangeMax: model.config().cell.width // Fixed unit width for non-ordinal
```
Comment (domoritz) [inline src/compile/scale.ts]: unit

**Comment by domoritz** [comment]
> Done

Almost

**Comment by kanitw** [comment]
Ok.  Hopefully done now.  If not, please just fix them for me and merge.

**Comment by domoritz** [comment]
Done

**Pull request merged**